packages/in_app_purchase/in_app_purchase/README.md

```diff
@@ -3,6 +3,13 @@
 A Flutter plugin for in-app purchases. Exposes APIs for making in-app purchases
 through the App Store (on iOS) and Google Play (on Android).
 
+<p align="left">
```

Comment:
@jayoung-lee I have added videos here, the url is set on master, which will start work after landing the PR. 

To test it, check out my repo locally, update the url to `https://github.com/flutter/cyanglaz/blob/iap_readme/packages/in_app_purchase/in_app_purchase/doc/iap_android.gif?raw=true`, and view the readme locally.

Reply:
I could not access the local repo with the URL. Is the repo private?